### PR TITLE
[EDA] Working EDA TSO

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
`cci flow run trial_org --org release`

### CODE REVIEW

File: cumulusci.yml [hunk 1]
```diff
@@ -114,6 +114,42 @@ tasks:
                 STG_InstallScript.enablePreferredPhoneSync();
                 STG_InstallScript.enablePreventDeletionHierarchyCustomSettings(NULL);
 
+    execute_automatic_household_naming:
+        description: Runs anonymous apex for dev to configure settings, insert Affiliation mappings, insert Relationship lookups and insert default tdtm trigger handlers.
```
Comment (jstvz) [inline cumulusci.yml:118]: Please update the description to match this task's action.
```suggestion
        description: Runs anonymous apex to set Automatic_Household_Naming__c to true.
```

File: cumulusci.yml [hunk 2]
```diff
@@ -114,6 +114,42 @@ tasks:
...
+        class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
+        options:
+            apex: >
+                %%%NAMESPACE%%%Hierarchy_Settings__c hierSettings = %%%NAMESPACE%%%Hierarchy_Settings__c.getOrgDefaults();
+                hierSettings.%%%NAMESPACE%%%Automatic_Household_Naming__c = true;
+                update hierSettings;
+
+    execute_preferred_phone_selection:
+        description: Runs anonymous apex for dev to configure settings, insert Affiliation mappings, insert Relationship lookups and insert default tdtm trigger handlers.
```
Comment (jstvz) [inline cumulusci.yml:127]: Please update the description to match this task's action.
```suggestion
        description: Runs anonymous apex to set Preferred Phone to 'Other Phone'.
```

File: cumulusci.yml [hunk 3]
```diff
@@ -114,6 +114,42 @@ tasks:
...
+        class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
+        options:
+            apex: >
+                %%%NAMESPACE%%%Hierarchy_Settings__c hierSettings = %%%NAMESPACE%%%Hierarchy_Settings__c.getOrgDefaults();
+                hierSettings.%%%NAMESPACE%%%Preferred_Phone_Selection__c = 'Other Phone';
+                update hierSettings;
+
+    execute_enable_course_connections:
+        description: Runs anonymous apex for dev to configure settings, insert Affiliation mappings, insert Relationship lookups and insert default tdtm trigger handlers.
```
Comment (jstvz) [inline cumulusci.yml:136]: Please update the description to match this task's action.
```suggestion
        description: Runs anonymous apex to enable Course Connections.
```

File: cumulusci.yml [hunk 4]
```diff
@@ -114,6 +114,42 @@ tasks:
...
+        class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
+        options:
+            apex: >
+                %%%NAMESPACE%%%Hierarchy_Settings__c hierSettings = %%%NAMESPACE%%%Hierarchy_Settings__c.getOrgDefaults();
+                hierSettings.%%%NAMESPACE%%%Affiliation_Record_Type_Enforced__c = true;
+                update hierSettings;
+
+    execute_send_error_notifications:
+        description: Runs anonymous apex for dev to configure settings, insert Affiliation mappings, insert Relationship lookups and insert default tdtm trigger handlers.
```
Comment (jstvz) [inline cumulusci.yml:145]: Please update the description to match this task's action.
```suggestion
        description: Runs anonymous apex to enable Error Notifications.
```

**Review by jstvz — CHANGES_REQUESTED** [review]
Thanks. Changes requested:
- [x] Update descriptions for the new tasks
- [ ] Update documentation/automation.md

**Review by MatthewBlanski — APPROVED** [review]

**Review by jstvz — APPROVED** [review]

File: cumulusci.yml [hunk 5]
```diff
@@ -114,6 +114,42 @@ tasks:
...
+    execute_preferred_phone_selection:
+        description: Runs anonymous apex to set Preferred Phone to 'Other Phone'.
+        class_path: cumulusci.tasks.apex.anon.AnonymousApexTask
+        options:
+            apex: >
+                %%%NAMESPACE%%%Hierarchy_Settings__c hierSettings = %%%NAMESPACE%%%Hierarchy_Settings__c.getOrgDefaults();
+                hierSettings.%%%NAMESPACE%%%Preferred_Phone_Selection__c = 'Other Phone';
+                update hierSettings;
+
+    execute_enable_course_connections:
```
Comment (MatthewBlanski) [inline cumulusci.yml:135]: @jofsky This is enabling record type validation and not course connections.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]

**Review by spinkelman — DISMISSED** [review]
QA testing passes. Approved! Nice work Josh!

**Pull request merged**